kafka/producer/base.py

```diff
@@ -208,7 +230,7 @@ def _send_messages(self, topic, partition, *msg, **kwargs):
             try:
                 resp = self.client.send_produce_request([req], acks=self.req_acks,
                                                         timeout=self.ack_timeout)
-            except Exception:
+            except:
```

Comment:
don't change it to a "naked exception", keep as it was `except Exception`...
minimize changes